### PR TITLE
Fix broken links and update link checking docs

### PR DESCRIPTION
**Pull request by BigLep**
This is done in support of https://github.com/filecoin-project/filecoin-docs/issues/2319

### CODE REVIEW

**Comment by BigLep** [comment]
This is the branch protection rule I want to enable:
https://github.com/filecoin-project/filecoin-docs/settings/rules/2131088
github com_filecoin-project_filecoin-docs_settings_rules_2131088

The key callouts are:
For the main branch...
1. Require PRs
2. Require PRs to have one approval before merging
3. Require comments to be resolved before merging
4. Require linkChecker to pass before merging
5. Block force pushes

I will enable this once it gets approval

**Comment by BigLep** [comment]
@eshon and @lanzafame : I think this is ready for review.  Please look at https://github.com/filecoin-project/filecoin-docs/pull/2327#issuecomment-2397823190 as well.

File: README.md
Comment (lanzafame) [inline README.md:36]: I would add that raw links need backticks and not `<code>` blocks around them to work with the link checker.

**Review by lanzafame — CHANGES_REQUESTED** [review]

File: README.md
Comment (BigLep) [reply README.md]: @lanzafame : i think `<code>` block works too.  I have updated the README.

**Review by lanzafame — APPROVED** [review]

File: README.md
Comment (lanzafame) [reply README.md]: Ahh good to know

**Pull request merged**
